doc/source/skips/0-governance.rst

```diff
@@ -0,0 +1,154 @@
+.. _governance:
+
+===========================================
+scikit-image governance and decision-making
+===========================================
+
+The purpose of this document is to formalize the governance process used by the
+scikit-image project, to clarify how decisions are made and how the various
+elements of our community interact.
+
+This is a consensus-based community project. Anyone with an interest in the
+project can join the community, contribute to the project design, and
+participate in the decision making process. This document describes how that
+participation takes place, how to find consensus, and how deadlocks are
+avoided.
+
+Roles And Responsibilities
+==========================
+
+Contributors
+------------
+Contributors are community members who contribute in concrete ways to the
+project. Anyone can become a contributor, and contributions can take many forms
+– not only code – as detailed in the
+:ref:`contributors guide <howto_contribute>`.
+
+Core developers
+---------------
+Core developers are community members that have demonstrated continued
+commitment to the project through ongoing engagement with the community. They
+have shown they can be trusted to maintain scikit-image with care. Being a core
+developer allows contributors to more easily carry on with their project
+related activities by giving them direct access to the project’s repository and
+is represented as being an organization member on the scikit-image
+`GitHub organization <https://github.com/orgs/scikit-image/people>`_.
+Core developers are expected to review code contributions, can merge approved
+pull requests, can cast votes for and against merging a pull-request, and can
```

Comment:
Maybe something like:

> , are encouraged to share their options for and against merging a pull-request

Instead of "vote"

Reply:
@hmaarrfk but this is something that any community member can do.